DOCSP-33172 added meta description for top 250 pages in docs-mongodb-shell (#307)
mongodb/docs-mongodb-shell: source/crud.txt

   :values: reference

.. meta:: 

>    :description: CRUD operations create, read, update, and delete documents within a MongoDB collection.

Reviewer: ```suggestion
   :description: Use the MongoDB Shell to create, read, update, and delete documents in a MongoDB collection.
```
